.eslintrc.json

```diff
@@ -0,0 +1,26 @@
+{
+    "root": true,
+    "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+        "ecmaVersion": 6,
+        "sourceType": "module"
+    },
+    "plugins": [
+        "@typescript-eslint"
+    ],
+    "rules": {
+        "@typescript-eslint/naming-convention": "warn",
+        "@typescript-eslint/semi": "warn",
+        "curly": "warn",
+        "eqeqeq": "warn",
+        "no-throw-literal": "warn",
+        "semi": "off",
+        "no-mixed-spaces-and-tabs": "error",
+        "indent": ["error", 2]
```

Comment:
I think that it's better to use two spaces instead of tabs because the following organizations use two spaces:

> Two spaces, not longer and no tabs: Google, npm, Node.js, Idiomatic, Felix

https://eslint.org/docs/latest/rules/indent